app/routes/pages/components/LandingPage.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { DisplayVisionShort } from './DisplayVision';
+import { Content } from 'app/components/Content';
+import TextWithRedTitle, { TextWithTitle } from 'app/components/TextWithTitle';
+import Statistic from 'app/components/Statistic';
+import EmailItem from 'app/components/EmailItem';
+import Icon from 'app/components/Icon';
+import styles from './LandingPage.css';
+import banner from 'app/assets/about-us-banner.png';
+
+const LandingPage = ({
+  whoWeAre,
+  whatWeDo,
+  whyWeDoIt,
+  postAddress,
+  officeHours,
+  officeAddress,
+  webkomOfficeAddress,
+  organizationNo
+}) => {
+  return (
+    <Content className={styles.pageContainer}>
+      <Link to={'/pages/info/17-strategidokument'}>-> Vision page</Link>
+      <a href="#contact" className={styles.contactUsLink}>
+        Kontakt oss
+      </a>
+      <img
+        className={styles.banner}
+        src={banner}
+        alt="Abakus - Linjeforeningen for Datateknologi og Kommunikasjonsteknologi ved NTNU"
+      />
+
+      <div className={styles.whoWhatWhyContainer}>
```

Comment:
We do have a set of components for basic flex containers and content, `app/components/Layout` and `app/components/Content` not sure if you considered using those?